### DISCUSSION TITLE
Kaliton

### DISCUSSION THREAD

**Discussion by iptoux** [dnschneid/crouton#4802] [Show and tell]
Hey folks, i'm using crouton on my HP Chromebook (AMD) and I'm also an friend of Cybersecurity (pentesting) so I like Kali Linux a lot.

A few weeks ago, i would try Kali Linux with crouton and run on an error. Because I had experience in coding/development i decide to fix the error[#4792](https://github.com/dnschneid/crouton/issues/4792). In this time a new error raised, so I fixed it too. 

In the end I was not happy with my Kali Linux install because it was to "lightweight" no tools, no options which variant of Kali i want. Yeah.... Your are right, i also coded this to my needs.

Actually I can install Kali Linux rolling with
`Crouton -t headless`

There are also core,default available (full, everything is in planning). The install using original Kali bootstrap and runs fine to end. I'm now working on some fixes and some recode/renaming. 

If you are interested on my fork/project give an response and I will open the repo.

Because I stripped down the original crouton (removed not needed stuff, merge targets, only Kali Linux as release!) i don't want to call it crouton, so ppl don't get confused. Kali + Crouton = Kaliton.

- Kali Linux rolling
- core, headless, default, full, everything 
- targets will be install menu metapackages (soon).

https://github.com/iptoux/kaliton

**Comment by DennisLfromGA** [comment]
@iptoux,

This leads to a dead link, 404 error: [https://github.com/iptoux/kaliton](https://github.com/iptoux/kaliton)

And there is no 'kaliton' repo listed here either: [https://github.com/iptoux?tab=repositories](https://github.com/iptoux?tab=repositories)

-**DennisLfromGA**

**Reply by iptoux** [reply]
Hey, this is because my repos is still "private" I'm on mobile and can't switch to public now. Will do this in around 90 minutes when I'm home. Please be patient. **PLEASE LOOK IN DEV BRANCH! (When public)**
I opened the post to get feedback if there is any need of such fork/repo for public, because when I make it public I had to give support/time to the project.

Contributors are welcome.

**Reply by iptoux** [reply]
@DennisLfromGA 
Okay, it's public now, please use dev branche, i had done no pull to main yet (still some tests needed).

**Comment by TomTravis** [comment]
so what do you do ?, run chroot headless and connect to it via shell? very cool

I have a similar setup Im using lenvo I9 server bookworm and https://wiki.x2go.org/doku.php wireless Dhcp

have arm chromebooks bullseye (mr chromebox) using X2go  clients to connect it runs super good vscode and chromium dev on the server

**Reply by iptoux** [reply]
It's an **kali linux (rolling)** based on **crouton**, rewriten to only use kali, the targets are a few of the [meta-packages](https://www.kali.org/docs/general-use/metapackages/) from original kali linux.

[Kaliton](https://github.com/iptoux/kaliton) has it's own starter, called **kali**. The installer will be recalled [kaliton](https://github.com/iptoux/kaliton) in the near future. Many things are in my "todo" and there are also [bugs](https://github.com/iptoux/kaliton/issues), but you can still use the "_alpha_" in my [main branch](https://github.com/iptoux/kaliton).

**Currently the following targets are included:**

starting an VT-cli (original was start_cli)
- core -> kali-linux-core
- default -> kali-linux-default
- headless -> kali-linux-headless

starting (should do) xfce desktop
- desktop (xfce) -> kali-desktop-xfce

**Comment by TomTravis** [comment]
you can do you own debootstrap for kali and create a chroot where you like and shell to it
if VT--cli works for you great  
i use mate desktop and debian bookwork
Try Mrchromebox tech, it will brick the machine is not careful still cannot get google flex installed
I use vscode instead of VT-cli on x2go  good luck
My Samsung chrome books still have the WPT screw on the MB to write a new boot rom

**Comment by TomTravis** [comment]
well how do i join 
I sent you a invite im using kali-rollling also
